.github/actions/get-test-plan-key/action.yml

```diff
@@ -27,20 +27,30 @@ runs:
         # Execute the GraphQL query and parse the results
         response=$(curl -H "Content-Type: application/json" -X POST -H "Authorization: Bearer ${{ inputs.xray_token }}" --data "$graphql_query" "https://xray.cloud.getxray.app/api/v2/graphql")
 
-        test_plans=$(echo "$response" | jq -c '.data.getTestPlans.results')
+        test_plans=$(echo "$response" | jq -c '.data.getTestPlans.results.[].jira')
 
-        test_plan_key=$(node - <<EOF
-        const data = JSON.parse('$test_plans');
-        let test_plan_key = '';
-        data.forEach(item => {
-            const key = item.jira.key;
-            const summary = item.jira.summary;
-            if (summary === "NIGHTLY OSS ${{ inputs.major_version }}") {
-              test_plan_key = key;
-            }
-        });
-        console.log(test_plan_key);
-        EOF)
+        # test_plan_key=$(node - <<EOF
+        # const data = JSON.parse('$test_plans');
+        # let test_plan_key = '';
+        # data.forEach(item => {
+        #     const key = item.jira.key;
+        #     const summary = item.jira.summary;
+        #     if (summary === "NIGHTLY OSS ${{ inputs.major_version }}") {
+        #       test_plan_key = key;
+        #     }
+        # });
+        # console.log(test_plan_key);
+        # EOF)
```

Comment:
obsolete lines should be removed. you will get them in the git log history if needed